axelor-production/src/main/java/com/axelor/apps/production/web/ManufOrderController.java

```diff
@@ -492,4 +494,16 @@ public void computeCostPrice(ActionRequest request, ActionResponse response) {
       TraceBackService.trace(response, e);
     }
   }
+
+  public void checkBillOfMaterialAndProdProcess(ActionRequest request, ActionResponse response) {
```

Comment:
Add this check in existing method which is called on the onClick event of planBtn and startBtn.
you can add this check in service.